packages/palette-docs/content/docs/elements/layout/Cards.mdx

```diff
@@ -2,42 +2,54 @@
 name: Cards
 ---
 
+<Playground title="Small Card with one image">
+  <div style={{ width: '50%' }}>
+    <SmallCard
+      images={[
+        "https://www.fillmurray.com/400/400",
+      ]}
+      title="This one has one image"
+      subtitle="It's good"
+    />
+  </div>
+</Playground>
+
 <Playground title="Small Card with two images">
-  <div style={{ width: 200 }}>
+  <div style={{ width: '50%' }}>
```

Comment:
I tried making the changes myself but you use your fork, and I have no idea how to add to that for a PR on the main repo.

Could we replace all the `<div style={{ width: '50%' }}>` with `<Box p="2" backgroundColor="orange" width="50%">`?
This way we can see the card inside that box a bit clearer? 🤔

Reply:
Ha:) I actually specifically changed the enclosed container to % so we can resize the page and see how the Card adapts. I think that will mimic more how it will be used in the wild. But I like your idea of coloring the boundary box. Will update in a bit!

Reply:
Yea, I noticed the `%`, and I thought it wasa  very nice touch :D